README.md

```diff
@@ -4,86 +4,42 @@
 
 Note, that this is a development version, alpha-stage.
 
-This repo would create a container with openshift-ansible fork and necessary tools to provision CentOS machines on GCP and 
+This repo would create a container with openshift-ansible fork and necessary tools to provision CentOS machines on GCP and
 use Ignition files to bootstrap the cluster.
 
-# Create `bootstrap.ign`
-Put your pull secret in `pull_secret.json` and run:
-```
-# Pull the latest installer image and create install configs
-sudo podman pull registry.svc.ci.openshift.org/openshift/origin-v4.0:installer
-sudo podman run --rm \
-  -v $(pwd):/output \
-  -e OPENSHIFT_INSTALL_PLATFORM="libvirt" \
-  -e OPENSHIFT_INSTALL_LIBVIRT_URI="qemu+tcp://192.168.122.1/system" \
-  -e OPENSHIFT_INSTALL_LIBVIRT_IMAGE="file:///unused" \
-  -e OPENSHIFT_INSTALL_CLUSTER_NAME="<yourname>" \
-  -e OPENSHIFT_INSTALL_BASE_DOMAIN="origin-gce.dev.openshift.com" \
-  -e OPENSHIFT_INSTALL_EMAIL_ADDRESS="<yourname>@redhat.com" \
-  -e OPENSHIFT_INSTALL_PASSWORD="muchsecuritywow" \
-  -e OPENSHIFT_INSTALL_PULL_SECRET_PATH="/output/pull_secret.json" \
-  -ti registry.svc.ci.openshift.org/openshift/origin-v4.0:installer \
-  create install-config
-
-# Update install configs to set 3 masters and 3 workers
-sed -i "/master/{n;s/1/3/}" .openshift_install_state.json
-sed -i "/worker/{n;s/1/3/}" .openshift_install_state.json
-sed -i "/master/{n;n;s/1/3/}" install-config.yml
-sed -i "/worker/{n;n;s/1/3/}" install-config.yml
-
-# Create ignition configs
-sudo podman run --rm \              
-  -v $(pwd):/output \
-  -e OPENSHIFT_INSTALL_PLATFORM="libvirt" \
-  -e OPENSHIFT_INSTALL_LIBVIRT_URI="qemu+tcp://192.168.122.1/system" \
-  -e OPENSHIFT_INSTALL_CLUSTER_NAME="<yourname>" \      
-  -e OPENSHIFT_INSTALL_BASE_DOMAIN="origin-gce.dev.openshift.com" \
-  -e OPENSHIFT_INSTALL_EMAIL_ADDRESS="<yourname>@redhat.com" \              
-  -e OPENSHIFT_INSTALL_PASSWORD="muchsecuritywow" \        
-  -e OPENSHIFT_INSTALL_PULL_SECRET_PATH="/output/pull_secret.json" \
-  -ti registry.svc.ci.openshift.org/openshift/origin-v4.0:installer \
-  create ignition-configs
-```
-
-This would create three files - `bootstrap.ign`, `master.ign`, `worker.ign`
+Requirements:
+* `podman`
+* `oc`
+* Pull secret for images
+* GCP account credentials in a json
+* SSH private key
 
 # Prepare GCP creds
-Prepare a folder which consists the following files (see `example-injected` folder):
+Put your pull secret in `pull_secret.json`
+
+Prepare "injected/" directory which contains the following files (see `example-injected` folder):
 * `gce.json` - this is the JSON which GCP uses to authenticate. `libra.pem` should work
 * `ssh-privatekey` - private key which would be used to access instances
```

Comment:
Do we need to move this tip  ``libra.pem` should work ` to the next line  * `ssh-privatekey` ?

Reply:
Oh, correct. Fixed